pkgs/applications/misc/uefitool/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchurl, qmake, zip }:
+
+stdenv.mkDerivation rec {
+  name = "UEFITool-${version}";
+  version = "0.25.1";
+
+  src = fetchurl {
```

Comment:
This should be a `fetchzip`.

Reply:
… wait, why? it's a download for a `.tar.gz`

Reply:
`fetchzip` is for any archive, not just ZIP. GitHub may (and sometimes does) recompress revision archives, so this will break unless we checksum contents rather than archive itself.

Reply:
Yes. Doesn't mean it's not semantically the correct one, esp. when downloading a `.tar.gz` :)

Reply:
That's controversial and still up to debate. I actually prefer `fetchTarball`/`fetchzip` style interface over `fetchFromGitHub`. I agree `fetchzip` should be renamed.

See https://github.com/NixOS/nixpkgs/issues/32997#issuecomment-353725653, https://github.com/NixOS/nixpkgs/pull/27327#issuecomment-314797154, https://github.com/NixOS/nixpkgs/issues/32997#issuecomment-353702812.